packages/components/src/components/Badge/Badge.tsx

```diff
@@ -83,18 +83,18 @@ const getSizeProps = (
   if (size === "large") {
     return {
       fontSize: "fontSize30",
-      paddingBottom: "space30",
-      paddingLeft: "space50",
-      paddingRight: "space50",
-      paddingTop: "space25",
+      paddingBottom: "d2",
+      paddingLeft: "d4",
+      paddingRight: "d4",
+      paddingTop: "d1_5",
```

Comment:
I couldn't find `d1_5` in [storybook](https://pluto-storybook.localyze.com/?path=/docs/docs-spacing--docs). It exists but it's missing there?

Reply:
The storybook table does not parse some values correctly, omitting some tokens but [it exists](https://github.com/Localitos/pluto/blob/main/packages/theme/src/__tests__/__snapshots__/theme.test.tsx.snap#L939). Good looking out!